### ISSUE TITLE
.remove() does not allow field removal

### ISSUE THREAD

**Issue by petervavro** [orientechnologies/orientjs#358]
Sorry guys, this seems to be done but I can not get it to work even in v3.0.3. Please give me a hint why. Thank you !

_Originally posted by @petervavro in https://github.com/orientechnologies/orientjs/issues/267#issuecomment-457567158_

**Comment by petervavro** [comment]
Hi guys, any updates on this please ?

**Comment by wolf4ood** [comment]
Hi @petervavro 

do you have a test snippet to reproduce this issue?

**Comment by dargolith** [comment]
Hi!

I have the same problem with using .remove() to remove a property.

Try this on localhost db test with credentials root / root.
```
const OrientDBClient = require('orientjs').OrientDBClient;

OrientDBClient.connect({host: 'localhost', port: 2424}).then(client => client.session({username: 'root', password: 'root', name:
'test'}).then(session => {
console.log(session.update('V').remove('someProp').buildStatement());
client.close();}));
```

Output:
`UPDATE V REMOVE `

Expected:
`UPDATE V REMOVE someProp`

Am I using it wrong? Another question, if I want to remove several properties, what is the syntax? Calling it several times? Using an array?

Appreciate any help. :)

**Comment by wolf4ood** [comment]
Hi @petervavro  @dargolith 

i've just pushed ad fix and released OrientJS 3.0.4.

@dargolith 
you should use multiple remove calls
